### ISSUE TITLE
Charging Pad Not Receiving Power

### ISSUE THREAD

**Issue by AngellusMortis** [Myrathi/ChargePadsEx#9]
I am using your 3.0 experimental version. I am using IC2 2.0.150. Also have GregTech and GraviSuite. The charging pads are not receiving power (they are set up properly).

**Comment by jjjworley** [comment]
Same here, but no GregTech and using IC2-216

**Comment by Myrathi** [comment]
Persistent issue until I get back from holidays and fix it. Thanks for the update regarding 216.

**Comment by Zyles** [comment]
Have this issue too. Can't get power to charge pad. Use IC2 exp and Gregtech (latest of both).

**Comment by Glebiys** [comment]
Author, your mod is awesome! So many servers are using it. Why don't you fix that only problem?
Maybe you think that nobody need that or not interested in mod, but you are wrong!
Tons of players want to see this mod working on their servers.
So please, fix it. Specially you already promised to do that 5 months ago.

Please answer. Even if you are not planing to fix the issue. Just let us know.

Thank you.

P.S. issue is the same. Pads don't receive power. Using ChargePads v3.0.0.94, IC2 exp 2.0.357, BuildCraft 4.2.1

**Comment by rsg888** [comment]
Please, fix this bug or if don`t  have time for developing this mod grant rights to another developer!
Sorry for my english, im from Russia.

**Comment by Glebiys** [comment]
Author doesn't give a shit ab this mod anymore. So forget about it...

Автору насрать на этот мод уже давным давно. Так что забудь...

**Comment by Myrathi** [comment]
Actually, I'm currently waiting on new IC2 builds from Thunderdark to see if the powernet has improved enough to warrant me "giving a shit" again.

IC2 started moving in a direction that completely and totally ruined the play experience for me (ie, it back-pedalled and started going all GregTech), which meant I stopped using the mod, which is why I haven't updated ChP in a significant period of time.

As for whether the new builds with Thunderdark's changes will change my mind, again? Time will tell. No promises, though.

**Comment by rsg888** [comment]
(just a note) in lastest versions of ic2 for 1.7.10 was added a charge pad

**Comment by Myrathi** [comment]
Well, if that's true, that saves me the trouble. ;]

**Comment by florian96** [comment]
Do you have Fixed the issue for 1.6.4? I use GregTech with IC 2

**Comment by jjjworley** [comment]
Hi Florian

```
            I don't think they fixed the charging pad mod, but the new Industrialcraft 2 experimental mod has the charging pads built into the mod. They work really well.
```

**Comment by Myrathi** [comment]
@florian96 @jjjworley Indeed. Thunderdark asked if I minded him adding them directly and, honestly, I had no issues with it. They don't quite work the same (or as well, IMO), but they're there. :P
